client/my-sites/checkout/composite-checkout/components/item-variation-picker/variant-dropdown-price.tsx

```diff
@@ -206,13 +207,17 @@ export const ItemVariantDropDownPrice: FunctionComponent< {
 		} else if ( introCount > 1 ) {
 			if ( productBillingTermInMonths > 12 ) {
 				return introTerm === 'month'
-					? translate(
-							'%(formattedCurrentPrice)s first %(introCount)s months then %(formattedPriceBeforeDiscounts)s per %(billingTermInYears)s years',
-							{ args }
+					? preventWidows(
+							translate(
+								'%(formattedCurrentPrice)s first %(introCount)s months then %(formattedPriceBeforeDiscounts)s per %(billingTermInYears)s years',
+								{ args }
+							)
 					  )
-					: translate(
-							'%(formattedCurrentPrice)s for first %(introCount)s years then %(formattedPriceBeforeDiscounts)s per %(billingTermInYears)s years',
-							{ args }
+					: preventWidows(
+							translate(
+								'%(formattedCurrentPrice)s for first %(introCount)s years then %(formattedPriceBeforeDiscounts)s per %(billingTermInYears)s years',
+								{ args }
+							)
```

Comment:
I think we could wrap the whole expression altogether with `preventWidows()` but I'm also okay with keeping it like this. Great to reuse this function btw!